src/apps/search-header/search-header.test.ts

```diff
@@ -0,0 +1,213 @@
+describe("Search header app", () => {
+  beforeEach(() => {
```

Comment:
There is a lot of mock data here. You could consider move the data to a fixture like here: https://github.com/danskernesdigitalebibliotek/dpl-react/pull/83/files#diff-2681eddd9ccddb0757dced492f1e67c5065af0d811d13c6d071a7b9d09124753R86

Reply:
Diddly done :)